class LegateError(Exception):
    pass


class CheckError(LegateError):
    pass
